### PR TITLE
Improve RuboCop setup

### PR DESCRIPTION
**Pull request by mokagio**
### Description

Nothing but some RuboCop-guided tweaks.

### Testing instructions

Nothing to test per se, but worth checking the diff just in case something broken syntax wise.

### Screenshots

N.A.
---

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. — N.A.

### CODE REVIEW

File: fastlane/Deliverfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ app_identifier 'com.automattic.woocommerce'
 screenshots_path './fastlane/promo_screenshots/'
 
 # Make sure to update these keys for a new version
-app_version "9.9"
+app_version '9.9'
```
Comment (mokagio) [inline fastlane/Deliverfile]: Really, it's about time we remove the `deliver` setup from this repo, like we did in WordPress.

File: fastlane/Fastfile [hunk 2]
```diff
@@ -127,17 +129,12 @@ platform :ios do
...
-  ENV[GHHELPER_REPO = 'woocommerce/woocommerce-iOS']
   ENV['PROJECT_NAME'] = 'WooCommerce'
   ENV['PUBLIC_CONFIG_FILE'] = 'config/Version.Public.xcconfig'
   ENV['PROJECT_ROOT_FOLDER'] = './'
   ENV['APP_STORE_STRINGS_FILE_NAME'] = 'AppStoreStrings.pot'
   ENV['FL_RELEASE_TOOLKIT_DEFAULT_BRANCH'] = 'trunk'
 
-  # Other defines used across multiple lanes
-  REPOSITORY_NAME = 'woocommerce-ios'
-  GH_ORG_NAME = 'woocommerce'
```
Comment (mokagio) [inline fastlane/Fastfile:139]: These were actually unused.

**Comment by mokagio** [comment]
@crazytonyli for some reason I can't find you in the reviewers box, so here's a direct ping. Thank you for your time 🙇‍♂️

**Review by crazytonyli — APPROVED** [review]

**Comment by crazytonyli** [comment]
I think I can't be a reviewer because I am not in a group with write access. I can't merge this PR too.

@jkmassel Do you mind having a look if I'm in the right groups of this GitHub org?

**Comment by mokagio** [comment]
Not sure why auto-merge hasn't kicked in 🤔 The PR has been approved and CI is green...

image

I'll disable and merge manually.

---

Update: Just had to look closer... 👀 

The problem was with @crazytonyli's user access to the org, which is why the tick which usually is green is now green.

**Review by crazytonyli — APPROVED** [review]

**Comment by mokagio** [comment]
Still not working, grey tick...

image

But @crazytonyli, you are a member of the team...

image

**Comment by mokagio** [comment]
Ah! This was the issues I think... The new team had "Read" access to the repo, which was not enough to approve PRs:

image

**Pull request merged**